docs/guide/template-filters.md

```diff
@@ -56,63 +61,71 @@ module.exports = {
   }
 };
 ```
-To use this filter you would simply pipe your data to the filter.
 
+Within the `methods`, we create a function `stripHttp()` that takes a string from the template. It performs a RegExp replace to strip the protocol before returning the string.
+
+In `init()`, we pass our function to the template module using `self.apos.template.addFilter({})`. This method takes the name of the filter that will be used in the template as property - in this case `stripHttp` - and our new function as a value.
+
+To use this new filter you would simply pipe your data to the filter from within the template.
 ```markup
 // lib/modules/link-widgets/views/widget.html
 <section data-link-widget>
-  <h2>{{ data.widget.label }}: {{ data.widget.url | stripHttp }}</h2>
+  <a href="{{ data.widget.url }}">{{ data.widget.label }}: {{ data.widget.url | stripHttp }}</a>
 </section>
 ```
 
 ## Alphabetical Apostrophe filter reference
+### `| build(url, path, data)`
 
-### `| build(obj...)`
+This filter can add path and query string parameters to the passed URL. This is very useful for adding filters to the current URL, respecting other filters already present, without complicated logic.
+The method in the `url` module requires that the URL be passed in as the first argument. However, when using this as a filter in Nunjucks, the data before the pipe will be sent as the first argument. This parameter accepts a URL that can include query parameters and anchor tags.
 
-Given a URL and one or more objects, this filter adds query string parameters for each of the properties in the objects. If objects affect the same parameter, the last object wins. If a parameter is set to `null` or the empty string it is removed from the URL altogether. This is very useful for adding filters to the current URL, respecting other filters already present, without complicated logic.
+The `path` parameter accepts an array but is optional. If present, it allows the addition of additional path elements to the URL. The array strings are not added directly to the URL but are substituted with values from the `data` object. So, with a `path` array of `['one', 'two']` and a `data` object of `{ one: 'first', two: 'second'}` the URL `https://example.com` would be modified to `https://example.com/first/second`. The path elements will be added in the order they appear in the `path` array.
```

Comment:
The `path` parameter is working - maybe you were thinking of the `order` key of the admin-bar module? I modified the text to indicate that it can take multiple `data` argument objects. Embarrassingly, I forgot that you could refer to the passed arguments as `arguments` and not be limited to the number of stated parameters.